docs/modules/ROOT/pages/explanations/commodore-components/crds.adoc

```diff
@@ -0,0 +1,86 @@
+= Installing CRDs in Commodore components
+
+Commodore components may introduce new custom resources, and should do so when installing operators that depend on them.
+A component needs to be careful that the provided CRDs are always compatible with the installed operator.
+
+One needs to take special considerations when support for a CRD version is dropped.
+When this happens all existing objects https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version[need to be upgraded to a new stored version.]
+This is often not considered a breaking change by the upstream project.
+The component needs to either handle this stored version upgrade in a job or treat such an update as a breaking change and provide an upgrade guide to do so.
+
+It's considered best practice to provide ClusteRoles that https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles[are aggregated] to the `view`,  `edit`, `admin`, and if applicable `cluster-reader` ClusteRole.
+This is done by setting the label `rbac.authorization.k8s.io/aggregate-to-<role>` on the provided ClusteRoles.
+
+As the names suggest you should:
+
+* Add get, list, and watch permissions for all namespaced custom resources to `view`
+* Add create, update, and delete permissions for all namespaced custom resources to `edit`
+* Add permissions for sensitive custom resources to `admin`
+* Add get, list, and watch permissions for all cluster scoped custom resources to `cluster-reader`
+
+== Example
+
+In this example we look at a component that installs two custom reasource definitions.
+The namespaced `foos.foo.example.com` resource, and the cluster-wide `clusterfoos.foo.example.com` resource.
```

Comment:
This isn't a full sentence (no verb) -> either use my suggestion of adding a colon instead of a period in the line above it (that fixes the issue), or reword it somehow.